src/features/jserrors/instrument/index.js

```diff
@@ -124,6 +124,8 @@ function notice (err, doNotStamp, ee) {
   // this is to preserve existing behavior
   var time = (!doNotStamp) ? now() : null
   handle('err', [err, time], undefined, FEATURE_NAMES.jserrors, ee)
+  // emit to session replay feature if present
+  handle('err', [err, time], undefined, FEATURE_NAMES.sessionReplay, ee)
```

Comment:
Just wondering if we are expecting any alteration of the err object by the session replay feature like the spa feature does.

Reply:
this is a good question.  In one way or another, we will need to decorate the error object that gets harvested with a flag indicating a session recording was occurring when the error was captured. This could be done in a contextual way like mentioned above, or by checking the session manager at harvest time in the error feature

Reply:
We should look for a way to do this "decorating" across features without creating dependency between features. I'm looking at you spa feature!!!

Reply:
agreed, changed to use the errorAgg call to match the others